misk/src/main/kotlin/misk/web/actions/reflect.kt

```diff
@@ -61,3 +62,45 @@ internal fun Class<*>.superclasses(): Set<Class<*>> {
   }
   return result
 }
+
+/**
+ * Attempts to find a non-synthetic method with the same declaring class, name, and parameters. If
```

Comment:
Thanks for the docs, helps to understand the purpose (and flakiness without) this code!